### PR TITLE
Add bet365 API

### PR DESCRIPTION
**Pull request by vaishakhp1902**
- [✅] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [✅] My addition is ordered alphabetically
- [✅] My submission has a useful description
- [✅] The description does not have more than 100 characters
- [✅] The description does not end with punctuation
- [✅] Each table column is padded with one space on either side
- [✅] I have searched the repository for any relevant issues or pull requests
- [✅] Any category I am creating has the minimum requirement of 3 items
- [✅] All changes have been [squashed][squash-link] into a single commit

### CODE REVIEW

**Review by pawelborkar — CHANGES_REQUESTED** [review]
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

File: README.md [hunk 1]
```diff
@@ -1470,6 +1470,7 @@ API | Description | Auth | HTTPS | CORS |
 | [API-FOOTBALL](https://www.api-football.com/documentation-v3) | Get information about Football Leagues & Cups | `apiKey` | Yes | Yes |
 | [ApiMedic](https://apimedic.com/) | ApiMedic offers a medical symptom checker API primarily for patients | `apiKey` | Yes | Unknown |
 | [balldontlie](https://balldontlie.io) | Balldontlie provides access to stats data from the NBA | No | Yes | Yes |
+| [Bet365](https://rapidapi.com/b365api-b365api-default/api/betsapi2/) | Live results and fixtures on a variety of sports for betting | `apiKey` | Yes | Unknown |
```
Comment (pawelborkar) [inline README.md:1473]: The API is mainly returning the responses related to events. 
So It's better to update the description according to it and please don't use the words like betting and fixture. Please polish the description a bit.

Thanks

**Review by pawelborkar — CHANGES_REQUESTED** [review]
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

**Comment by pawelborkar** [comment]
Hey @vaishakhp1902,
Can you commit the suggested changes today only?
As we are very close to our milestone.
Please don't forget to squash commits into single one and if you don't know how to do it then you can open a new PR and close this one,

Thanks

**Comment by vaishakhp1902** [comment]
Hey @pawelborkar 
Thanks for your reply. I'm pretty new to this, I messed up my branch and was not able to squash, ill close this and open a new one.

Thanks

**Comment by pawelborkar** [comment]
Sure 😊

**Pull request closed without merging**